js-packages/@prestojs/viewmodel/src/fields/Field.ts

```diff
@@ -67,7 +67,7 @@ class UnboundFieldError<T, K> extends Error {
  * @extract-docs
  * @menu-group Fields
  */
-export default class Field<T, ParsableType extends any = T> {
+export default class Field<T, ParsableType extends any = T, SingleType = T> {
```

Comment:
Is it documented anywhere what ParsableType is?

Reply:
No generally haven't documented generic types, will add something